src/authorization/utils.ts

```diff
@@ -94,3 +95,12 @@ export const cancelAxiosRequestAndMakeFetch = (
     })
   };
 };
+
+export const validateAuthType = (authType: string): boolean => {
+  const schema = object().shape({
+    email: string().email().required()
+  });
+  const input = { email: authType };
+  const isValid = schema.isValidSync(input);
+  return isValid;
+};
```

Comment:
if this is just validating that a user id is an email, then I think we should rename the function to `isEmail` and probably could just do a regex check to validate the email.